frame/contracts/src/migration.rs

```diff
@@ -249,3 +255,22 @@ mod v6 {
 		weight
 	}
 }
+
+/// Rename `AccountCounter` to `Nonce`.
+mod v7 {
+	use super::*;
+
+	pub fn migrate<T: Config>() -> Weight {
+		generate_storage_alias!(
+			Contracts,
+			AccountCounter => Value<u64, ValueQuery>
+		);
+		generate_storage_alias!(
+			Contracts,
+			Nonce => Value<u64, ValueQuery>
+		);
+		Nonce::set(AccountCounter::take());
+		T::DbWeight::get().reads_writes(1, 2)
```

Comment:
Where's the second write here? `take()` a read + a clear, so does a clear count as a write?

Reply:
Yes the removal of a value counts as write.